src/System.Console/tests/System.Console.Tests.csproj

```diff
@@ -27,6 +27,9 @@
     <Compile Include="ThreadSafety.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="TermInfo.cs" />
+  </ItemGroup>
```

Comment:
Agreed. I'll switch the tests over to use reflection to invoke the methods rather than InternalsVisibleTo; I'll do the switch tomorrow morning before I go OOF